corehq/apps/locations/models.py

```diff
@@ -77,6 +77,7 @@ class LocationType(models.Model):
     administrative = models.BooleanField(default=False)
     shares_cases = models.BooleanField(default=False)
     view_descendants = models.BooleanField(default=False)
+    last_modified = models.DateTimeField(auto_now=True)
```

Comment:
Seems reasonable to ping @dannyroberts just in case.  Danny, any objections to using `models.DateTimeField(auto_now=True)`?  Does it attach tz information or is it safe to store with the assumption that it's always UTC?